fs_storage/models/fs_storage.py

```diff
@@ -363,6 +375,12 @@ def _get_filesystem(self) -> fsspec.AbstractFileSystem:
         directory_path = self.directory_path
         if directory_path:
             fs = fsspec.filesystem("rooted_dir", path=directory_path, fs=fs)
+        if self.invalidate_cache_on_error:
+            # Check whether we need to invalidate cache or not
+            try:
+                fs.ls("", detail=False)
```

Comment:
If everything is stored at the root of the file system, this will shortly introduce a performance issue. Why not `fs.exists("\")`?

Reply:
@lmignon true, I was using the same approach as in the method to test the configuration (not sure if maybe we could also change it there)

Reply:
> @lmignon true, I was using the same approach as in the method to test the configuration (not sure if maybe we could also change it there)

IMO we should

Reply:
@lmignon changed :)